src/runtime/virtcontainers/qemu.go

```diff
@@ -363,19 +378,36 @@ func (q *qemu) createQmpSocket() ([]govmmQemu.QMPSocket, error) {
 		return nil, err
 	}
 
+	qsks, err := q.qmpSocketPaths(q.id)
+        if err != nil {
+                return nil, err
+        }
+
 	q.qmpMonitorCh = qmpChannel{
 		ctx:  q.ctx,
 		path: monitorSockPath,
 	}
 
-	return []govmmQemu.QMPSocket{
-		{
-			Type:   "unix",
-			Name:   q.qmpMonitorCh.path,
-			Server: true,
-			NoWait: true,
-		},
-	}, nil
+	qmpSks := []govmmQemu.QMPSocket{
```

Comment:
This can be simplied by:

```golang
qsks = append(qsks, q.qmpMonitorCh.path)

for _, s := range qsks {
                qmpSks = append(qmpSks,
                        govmmQemu.QMPSocket{
                                Type:   "unix",
                                Name:   s,
                                Server: true,
                                NoWait: true,
                        })
        }
```

Reply:
change qmpSocketPath to qmpSocketPaths. default qmp.sock is the first one

Reply:
deleted qmpSocketPath because qmpSocketPath is like qmpSocketPaths.
origin qmp.sock is the first one of returned value of qmpSocketPaths.

Reply:
Then you can reverse the order.